Cargo.toml

```diff
@@ -15,7 +15,8 @@ members = [
     "examples/experimentation_client_integration_example",
     "examples/cac_client_integration_example",
     "examples/superposition-demo-app",
-    "crates/superposition_macros"
+    "crates/superposition_macros",
+    "crates/superposition_proc_macros"
```

Comment:
Can we call this crate superposition_derives? That seems to be the conventional way

Reply:
Sure